src/lib/core/ripple/index.ts

```diff
@@ -6,7 +6,7 @@ import {SCROLL_DISPATCHER_PROVIDER} from '../overlay/scroll/scroll-dispatcher';
 
 export {MdRipple, RippleGlobalOptions, MD_RIPPLE_GLOBAL_OPTIONS} from './ripple';
 export {RippleRef, RippleState} from './ripple-ref';
-export {RippleConfig} from './ripple-renderer';
+export {RippleConfig, RIPPLE_FADE_IN_DURATION} from './ripple-renderer';
```

Comment:
also exported RIPPLE_FADE_OUT_DURATION in a separate commit. Let me know what you think!